java-checks/src/main/java/org/sonar/java/filters/SuppressWarningFilter.java

```diff
@@ -53,22 +50,22 @@
 
 public class SuppressWarningFilter extends BaseTreeVisitorIssueFilter {
 
-  private static final Multimap<String, String> JAVAC_WARNING_SUPPRESSING_RULES = new ImmutableMultimap.Builder<String, String>()
-      .put("cast", "java:S1905")
-      .put("deprecation", "java:S1874")
-      .put("dep-ann", "java:S1123")
-      .put("divzero", "java:S3518")
-      .putAll("empty", "java:S1116", "java:S108")
-      .put("fallthrough", "java:S128")
-      .put("finally", "java:S1143")
-      .put("overrides", "java:S1206")
-      .put("removal", "java:S5738")
-      .put("serial", "java:S2057")
-      .put("static", "java:S2209")
-      .put("rawtypes", "java:S3740")
+  private static final Map<String, Set<String>> JAVAC_WARNING_SUPPRESSING_RULES = MapBuilder.<String, Set<String>>newMap()
+      .put("cast", Collections.singleton("java:S1905"))
+      .put("deprecation", Collections.singleton("java:S1874"))
+      .put("dep-ann", Collections.singleton("java:S1123"))
+      .put("divzero", Collections.singleton("java:S3518"))
+      .put("empty", ImmutableSet.of("java:S1116", "java:S108"))
```

Comment:
Will remove it after PR with ImmutableSet is merged

Reply:
go go :)